client/state/happychat/selectors/is-precancellation-chat-available.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Returns if precancellation chat is available.
+ * @param   {Object}  state  Global state tree
+ * @returns {Boolean}        true, when precancellation is available
+ */
+export default function isPrecancellationChatAvailable( state ) {
+	return state.happychat.user.isPresalesPrecancellationEligible.precancellation;
```

Comment:
Since there is no guarantee that the substate `happyisPresalesPrecancellationEligible` isn't null from the reducer side, I'd suggest we use `lodash.get` to make this safer. e.g.

```
import { get} from 'lodash';
...
return get( state, 'happychat.user.isPresalesPrecancellationEligible.presale', false );
```

The same goes to `isPresaleChatAvailable()` below.

That way, we don't have to default the connected props from components, either.